components/Form/Form.scss

```diff
@@ -0,0 +1,86 @@
+.form {
+  padding: $section-padding-mobile 0;
+
+  &__wrapper {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__image {
+    display: none;
+    border-radius: 2.3rem;
```

Comment:
add all the rules (especially the image url) at the breakpoint when you need them, not earlier